src/autoqasm/converters/typecast.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+"""Converters for integer casting nodes."""
+
+import ast
+
+import gast
+from malt.core import ag_ctx, converter
+from malt.pyct import templates
+
+
+class TypecastTransformer(converter.Base):
+    def visit_Call(self, node: ast.stmt) -> ast.stmt:
+        """Converts type casting operations to their AutoQASM counterpart.
+
+        Args:
+            node (ast.stmt): AST node to transform.
+
+        Returns:
+            ast.stmt: Transformed node.
+        """
+        typecasts_supported = ["int", "float"]
+
+        template = """
+                ag__.typecast(type_, argument_)
+                """
```

Comment:
Should you remove the newlines from this template? If you're just replacing a function call, the newlines might cause syntax errors in the transpiled Python.

Reply:
Done!